samples/server/go-server/api/server/storage/memstore_test.go

```diff
@@ -29,7 +30,10 @@ import (
 
 func TestCreateNote(t *testing.T) {
 	s := NewMemStore()
-	n := testutil.Note()
+	ps := NewProjectMemStore()
```

Comment:
What is the purpose of creating ps and calling ps.CreateProject()? It doesn't appear that the function affects any global state that influences the test...

Reply:
Ahhh...good catch! Previous versions required there to be a project to be able to create notes/occurrences/operations. This is now enforced in the application layer instead.

Fixed in 7e350b5